csv_to_json_unlabelled.py

```diff
@@ -272,7 +272,7 @@ def build_examples(row: pandas.Series, utterance_col: str, convo_id_col: str = '
         external_id = f'example-{row[convo_id_col]}-{row["idx"]}'
         context = humanfirst.objects.HFContext(
             context_id=row[convo_id_col],
-            context_type='conversation',
+            type='conversation',
```

Comment:
Need not to revert this - the script will crash.   I think this was a merge error.  Humanfirst module expects context_type no?

Reply:
The schema expects type instead of context _type. Made the changes required in the humanfirst module.
![image](https://github.com/zia-ai/academy/assets/101306027/cca9c0a3-d090-43bc-bb66-10c3795ad202)

The script doesn't crash with the new updated humanfirst module.